lib/src/layout_adjustments/alignment.dart

```diff
@@ -0,0 +1,117 @@
+enum HorizontalAlignment {
```

Comment:
`Alignment` in SwiftUI can be defined in terms of `HorizontalAlignment` and `VerticalAlignment` so that's why I moved them to a common file rather than importing `HStack` and `VStack` into the `Frame` file.

The `layout_adjustments` already existed and for the `Frame` widget, so that's why I put `alignment.dart` there. That's sounds good to move `alignment.dart` to the `layout` directory. I assume I should move `frame.dart` there too?